newsfragments/24755.improvement.rst

```diff
@@ -0,0 +1 @@
+Log server on worker binds IPv6 interface.
```

Comment:
I think the number here is wrong? It should be PR number not issue number I believe

Reply:
Actually - It does not matter in the long run - it is important they are unique between merges. The newsfragements get deleted when release notes are prepared and the numbers are gone. In case of GitHub PR and Issue share the same number space (so there is never a PR with the same number that Issue). It's actually a bit easier to use Issue number than PR number (when you have an issue) because you actually KNOW it before you make a PR. Otherwsie the process of creating newsfragment is a bit cumbersome:

1) create a branch
2) make a draft PR
3) add newsframent and amand the PR (this is the first moment you know PR #)
4) push ameneded commit and un-draft PR

I think using issue id brings us no problems and helps with making the process simpler (when you have an issue)

1) create a branch with issue # newsfragment
2) create a PR

@jedcunningham  - wdyt?

Reply:
Yep, either works. It has to be unique, and it's used to build a link in the release notes, otherwise it really doesn't matter.

Reply:
BTW. In `pip` they even follow a bit more strict approach (but I actually started to like ours where we only add newsfragments when we really need it) - whenever there is an "insignificant" change - they have a semi-generated unique empty random number for newsfragment - so practically, the number we have for newsfragment does not have to be connected with issue or pr or anything else - it is enough that it is unique until it gets deleted at release time.

Reply:
> Yep, either works. It has to be unique, and it's used to build a link in the release notes, otherwise it really doesn't matter.

Comment: This I did not realise. We do not actually have to use it to build the link - we can get it from the actual PR# (which is in the subject) - I am not sure if it matters that much though - if we are cool with having sometimes link to issue and sometimes to PR, I am cool as well. And it does help to use newsfragment ID in case somoene (me again) forget about disabling "rebase & merge" :P

Reply:
> awesome - then maybe we just need to clarify it in the PR template

As usual @eladkal ... PRs are most welcome :P :P :P

Reply:
already in progress